src/web/utils/parserUtils.ts

```diff
@@ -117,7 +117,7 @@ function addTextToElement(node: TreeNode, text: string) {
   });
 }
 
-function addParagraph(node: TreeNode, text: string | null = null, length: number, disableInlineStyles = false) {
+function addParagraph(node: TreeNode, text: string | null, length: number, disableInlineStyles = false) {
```

Comment:
Why is this change necessary?

Reply:
We always pass this argument, so I figured out this `null` default value is redundant.